packages/core/src/Keymap.ts

```diff
@@ -17,13 +17,64 @@ interface ShortcutItem {
     priority: number;
 }
 
+/**
+ * Keymap allow to add and remove shortucts and provide a function to match if a
+ * particular pattern is found within registred shortucts.
+ *
+ * The expression to describe a shortuct is zero or more `modifiers` and one
+ * `hotkey` joined with the symbol `+`.
+ *
+ * ## Modifiers
+ * - SHIFT
+ * - ALT
+ * - CTRL
+ * - META
+ * - CMD (alias of META)
+ *
+ * Example:
+ * ```typescript
+ * // using a modifier and one key
+ * keymap.addShortcut('CTRL+A', 'a-command')
+ * // using multiples modifiers and one key
+ * keymap.addShortcut('CTRL+ALT+A', 'a-command')
+ * ```
+ *
+ * ## Hotkeys
+ * A hotkey can be wether a `key` or `code`.
+ *
+ * ### Key
+ * The syntax to describe a `key` is to write the `key` as it is. The convention
+ * is to write it in uppercase.
+ *
+ * Example:
+ * ```typescript
+ * keymap.addShortcut('A', 'a-command')
+ * ```
+ * The list of possible `key` values are defined in the following link:
+ * https://www.w3.org/TR/uievents/#dom-keyboardevent-key
+ *
+ * ### Code
+ * The syntax to describe a `code` is to write `<code>` (surrounded by "<" and
+ * ">").
+ *
+ * Example:
+ * ```typescript
+ * keymap.addShortcut('<KeyA>', 'a-command')
```

Comment:
I suggest adding an example with a modifier, as people might misunderstand and think they need to write `<CTRL+KeyA>` if they want to use a code, which would absolutely not work.